### PR TITLE
Add results for Node >=10.13 <11

### PR DESCRIPTION
**Pull request by Rowno**
Node 10.13 uses the same V8 version as Chrome 68 (V8 v6.8).

I figured it out by looking up Node's V8 version and then finding a Chrome release using a matching V8 version:

```sh
$ node --version
v10.13.0
```
```sh
❯ node -e "console.log(process.versions.v8)"
6.8.275.32-node.36
```

https://github.com/chromium/chromium/blob/68.0.3440.134/DEPS#L104

https://github.com/v8/v8/commit/e74ffb8d10fa5a8e29ae1187294a1c76346c712f

### CODE REVIEW

**Review by ljharb — APPROVED** [review]

File: environments.json [hunk 1]
```diff
@@ -2385,6 +2385,24 @@
       "non-standard"
     ]
   },
+  "node10": {
```
Comment (chicoxyzzy) [inline environments.json]: must be `"node10_13"` per our naming convention

**Review by chicoxyzzy — CHANGES_REQUESTED** [review]

**Comment by Rowno** [comment]
Fixed

**Review by chicoxyzzy — APPROVED** [review]

**Comment by afmenez** [comment]
We would have 4 different node versions active.  How about changing node 4 and 6 to obsolete?

**Comment by chicoxyzzy** [comment]
https://github.com/kangax/compat-table/pull/1288

**Pull request merged**